packages/datagateway-table/src/breadcrumbs.component.tsx

```diff
@@ -0,0 +1,538 @@
+import React, { useEffect } from 'react';
+import { StateType } from './state/app.types';
+import { connect } from 'react-redux';
+
+import axios from 'axios';
+import { Route } from 'react-router';
+import { Link } from 'react-router-dom';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+
+import {
+  Link as MaterialLink,
+  Paper,
+  Breadcrumbs,
+  Typography,
+  Tooltip,
+} from '@material-ui/core';
+import { TooltipProps } from '@material-ui/core/Tooltip';
+
+import {
+  withStyles,
+  StyleRules,
+  createStyles,
+  WithStyles,
+  makeStyles,
+  Theme,
+} from '@material-ui/core/styles';
+
+const arrowGenerator = (
+  color: string
+): Record<string, Record<string, string | number | Record<string, string>>> => {
+  return {
+    '&[x-placement*="bottom"] $arrow': {
+      top: 0,
+      left: 0,
+      marginTop: '-0.95em',
+      width: '2em',
+      height: '1em',
+      '&::before': {
+        borderWidth: '0 1em 1em 1em',
+        borderColor: `transparent transparent ${color} transparent`,
+      },
+    },
+    '&[x-placement*="top"] $arrow': {
+      bottom: 0,
+      left: 0,
+      marginBottom: '-0.95em',
+      width: '2em',
+      height: '1em',
+      '&::before': {
+        borderWidth: '1em 1em 0 1em',
+        borderColor: `${color} transparent transparent transparent`,
+      },
+    },
+    '&[x-placement*="right"] $arrow': {
+      left: 0,
+      marginLeft: '-0.95em',
+      height: '2em',
+      width: '1em',
+      '&::before': {
+        borderWidth: '1em 1em 1em 0',
+        borderColor: `transparent ${color} transparent transparent`,
+      },
+    },
+    '&[x-placement*="left"] $arrow': {
+      right: 0,
+      marginRight: '-0.95em',
+      height: '2em',
+      width: '1em',
+      '&::before': {
+        borderWidth: '1em 0 1em 1em',
+        borderColor: `transparent transparent transparent ${color}`,
+      },
+    },
+  };
+};
+
+const useStylesArrow = makeStyles((theme: Theme) =>
+  createStyles({
+    tooltip: {
+      position: 'relative',
+      backgroundColor: theme.palette.common.black,
+      fontSize: '0.875rem',
+    },
+    popper: arrowGenerator(theme.palette.common.black),
+    arrow: {
+      position: 'absolute',
+      fontSize: 6,
+      '&::before': {
+        content: '""',
+        margin: 'auto',
+        display: 'block',
+        width: 0,
+        height: 0,
+        borderStyle: 'solid',
+      },
+    },
+  })
+);
+
+const ArrowTooltip = (props: TooltipProps): React.ReactElement => {
```

Comment:
This can be extracted to a separate file

Reply:
@louise-davies addressed in https://github.com/ral-facilities/datagateway/pull/70/commits/cbb1b70b9953d05b1f7a235bcd8b622e7be25f87.